python/ray/experimental/client/server.py

```diff
@@ -45,8 +75,10 @@ def serve(connection_str):
 
 
 if __name__ == "__main__":
-    logging.basicConfig()
+    logging.basicConfig(level="INFO")
+    # TODO(barakmich): Perhaps wrap ray init
     ray.init()
+    set_client_api_as_ray()
```

Comment:
Do we need this explicit set? Seems we fall back to the real Ray anyways in the call wrappers.

Reply:
Yeah, I've thought about just making that happen by default, but I wanted a way to also error or log if I knew that I hadn't set the client API (until it became a problem in the worker). Probably fine to get rid of, but holding onto it as I figure out if workers can do any better (they may not be able to tell the difference)